src/apps/loan-list/list/loan-list.tsx

```diff
@@ -167,6 +177,29 @@ const LoanList: FC = () => {
     }
   }, [loans, openModalDueDate, open]);
 
+  useEffect(() => {
+    if (loans) {
+      if (view === "list") {
+        setDisplayedLoans(getSearchItems(loans, searchItemsShown));
+      } else {
+        const stackedLoans: LoanV2[] = getStackedSearchItems(
```

Comment:
It was not possible to use `UseGetStackedSearchItems`?
What happens if the service returns an error?